src/cpu/o3/rename.cc

```diff
@@ -145,7 +145,12 @@ Rename::RenameStats::RenameStats(statistics::Group *parent)
       ADD_STAT(tempSerializing, statistics::units::Count::get(),
                "count of temporary serializing insts renamed"),
       ADD_STAT(skidInsts, statistics::units::Count::get(),
-               "count of insts added to the skid buffer")
+               "count of insts added to the skid buffer"),
+      ADD_STAT(intWrites, statistics::units::Count::get(),
+               "count of registers freed and written back to integer free list"),
+      ADD_STAT(fpWrites, statistics::units::Count::get(),
+               "count of registers freed and written back to floating point free list")
```

Comment:
Either the stat names are confusing or the stat description is wrong. When I see "intWrites" I would assume that this means the number of times the integer register file was written. 

Maybe I could help more if you could explain the use of this statistic. Why would people want to know the number of times we've written the free list?

Could a better name be "freeListWritesInt" and "freeListWritesFP"?

Reply:
For a bit of context, @KNS3092 and I were interested in the number of times integer registers and float registers were freed and written back to the free list because we were developing a power model and found that the integer and float free lists have different power consumption associated with them (Even though there's just one free list that mixes integers and floats in gem5).
I believe the names you suggested describe what the stats are more fully.
We chose the original names because we wanted to be consistent with the "intLookups" and "fpLookups" stats that we used to track the number of reads for integers and floats from the free list.
We can definitely make that name change if you think it'll be easier for people to understand, or if we want to be more consistent with the above stats, maybe "intReturned" and "fpReturned" are good for describing the number of registers 'returned' to the free list?
Here's what they look like as they are now in the stats file:
```
system\cpu\rename\intLookups
system\cpu\rename\fpLookups
...
system\cpu\rename\intWrites
system\cpu\rename\fpWrites
```

Reply:
Got it. In that case, all of these names are confusing to me 😆 

I like "intReturned" an "fpReturned" better than simply "writes". That said, it's not a strong preference, so I'll leave it up to you.

Reply:
I agree with @powerjg about the naming updates.